index.bs

```diff
@@ -1100,6 +1100,15 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
             1. Return |constructCredentialAlg| and terminate this algorithm.
 
+        :   If the [=client=] concludes that it is impossible for any eligible [=authenticator=] to appear,
+        ::  The client MAY prompt the user for [=user consent|consent=] to create a new credential. If the user confirms [=user
+            consent|consent=], the [=client=] MAY return a {{DOMException}} whose name is "{{NotAllowedError}}" and terminate this
```

Comment:
I would write:
If the user confirms [=user consent|consent=]  ->  If the user [=user consent|consents=]